modules/apps/list-type/list-type-api/src/main/java/com/liferay/list/type/service/ListTypeEntryLocalService.java

```diff
@@ -102,7 +102,8 @@ public PersistedModel createPersistedModel(Serializable primaryKeyObj)
 	 * @return the list type entry that was removed
 	 */
 	@Indexable(type = IndexableType.DELETE)
-	public ListTypeEntry deleteListTypeEntry(ListTypeEntry listTypeEntry);
+	public ListTypeEntry deleteListTypeEntry(ListTypeEntry listTypeEntry)
```

Comment:
Can you separate code changed manually and code changed by buildService into different commits, please?